pwndbg/wrappers/checksec.py

```diff
@@ -1,30 +1,16 @@
 from __future__ import annotations
 
-from subprocess import CalledProcessError
+from pwnlib.elf import ELF
 
-import pwndbg.commands
-import pwndbg.lib.cache
-import pwndbg.wrappers
 
-cmd_name = "checksec"
-cmd_pwntools = ["pwn", "checksec"]
+def get_raw_out(local_path: str) -> str:
+    elf = ELF(local_path)
+    output = f"File: {elf.path}\n"
+    output += f"Arch: {elf.arch}\n"
+    output += elf.checksec()
+    return output
 
 
-@pwndbg.wrappers.OnlyWithCommand(cmd_name, cmd_pwntools)
-@pwndbg.lib.cache.cache_until("objfile")
-def get_raw_out(local_path: str) -> str:
```

Comment:
The change to `get_raw_out` breaks all other functions that use it.

It seems the `get_raw_out` gets the `get_raw_out.cmd` injected through the `OnlyWithCommand` decorator which is rather a debatable or overengineered design. We should probably refactor it in the future.

Reply:
Can you give an example of a function that breaks? I thought if the tests pass then it should work fine.

Reply:
Oh, I thought relro and pie status use a different CLI tool and so would not work. My bad.

Then this is fine.

I was also thinking that maybe we should have a helper func that would return a dict of all properties that we use here, but considering that `ELF.checksec()` has lots of code, then I don't think we want to repeat that.

OTOH, I still dislike that we check for string in output in relro/pie status. Those can be easily fetched via `e.relro` and `e.pie`.

Maybe we should:
1. Have a helper function to return a cached (through `pwndbg.lib.cache.cache_until(..)`) `ELF(...)` object (yeah I understand that then we will kinda leak the resource, but its fine)
2. Use it like `get_elf(path).checksec()`, `get_elf(path).relro` etc.

Reply:
@diivi ping, are u going to address above?